ctru-rs/src/services/romfs.rs

```diff
@@ -36,14 +63,19 @@ impl RomFS {
             },
             || {
                 let mount_name = CStr::from_bytes_with_nul(b"romfs\0").unwrap();
-                unsafe { ctru_sys::romfsUnmount(mount_name.as_ptr()) };
+                let _ = unsafe { ctru_sys::romfsUnmount(mount_name.as_ptr()) };
             },
         )?;
 
         Ok(Self { _service_handler })
     }
 }
 
+impl Drop for RomFS {
+    #[doc(alias = "romfsUnmount")]
+    fn drop(&mut self) {}
```

Comment:
By the way, implementing Drop like this actually does change the semantics of the `RomFS` struct. I don't think it should matter much in this case (mostly has implications [around lifetimes](https://doc.rust-lang.org/std/ops/trait.Drop.html#drop-check), I think), but just something to be aware of.

Edit: I guess since it contains a `ServiceReference` there's probably already some drop glue in place, so maybe it actually doesn't matter in this case.